src/platform/plugins/shared/discover/public/context_awareness/types.ts

```diff
@@ -228,6 +229,7 @@ export interface RowControlsExtensionParams {
    * Available actions for row controls
    */
   actions: {
+    openInNewTabExtPointAction?: (params: OpenInNewTabExtPointAction) => void;
```

Comment:
```suggestion
    openInNewTab?: (params: OpenInNewTabExtPointAction) => void;
```
For consumers, we can just call this `openInNewTab`.

We also don't need it in the row controls extension atm, and instead in `getChartSectionConfiguration`. We have a separate issue for it in #239036, so you can do it in a separate PR if you'd like to merge this first, but these are the steps:
- Create a new params interface for `getChartSectionConfiguration` like `ChartSectionConfigurationExtensionParams`.
- Add an `actions` object with the `updateESQLQuery` and `openInNewTab` actions.
- Find where `useProfileAccessor('getChartSectionConfiguration')` is used in code, and pass the params object with the actions (you can check how `updateESQLQuery` is passed to `getRowAdditionalLeadingControls` for reference).